src/webglimpse/label.ts

```diff
@@ -148,10 +148,20 @@ export function newLabelPainter(label: Label, xFrac: number, yFrac: number, xAnc
             gl.clear(GL.COLOR_BUFFER_BIT);
         }
 
-        const texture = label.texture;
-        if (texture) {
+        if (truncateLabel) {
+            const xRight = viewport.w - 2 / viewport.w;
```

Comment:
Where does `2` come from? Does this still function correctly if the `rowLabelInsets` are modified?